eras/allegra/impl/src/Cardano/Ledger/Allegra/Translation.hs

```diff
@@ -125,11 +125,11 @@ instance Crypto c => TranslateEra (AllegraEra c) UTxOState where
   translateEra ctxt us =
     return
       UTxOState
-        { utxosUtxo = translateEra' ctxt $ utxosUtxo us
-        , utxosDeposited = utxosDeposited us
-        , utxosFees = utxosFees us
-        , utxosPpups = translateEra' ctxt $ utxosPpups us
-        , utxosStakeDistr = utxosStakeDistr us
+        { sutxosUtxo = translateEra' ctxt $ sutxosUtxo us
```

Comment:
maybe `sUtxosUtxo` would be clearer?

Reply:
Nah, that would go against the convention we use everywhere that the prefix is the lowercase abbreviation of the type itself